onnxruntime/python/tools/transformers/models/llama/llama_inputs.py

```diff
@@ -171,6 +172,58 @@ def get_merged_sample_with_past_kv_inputs(
 
     return inputs
 
+def get_dynamo_inputs(
+    config: AutoConfig,
+    device: torch.device,
```

Comment:
This function looks to be the same as `get_merged_sample_with_past_kv_inputs`. Can you delete this function and call `get_merged_sample_with_past_kv_inputs` instead?

Reply:
Okay, got it! Thanks